projects/amazon_product_page/index.html

```diff
@@ -6,11 +6,32 @@
     <link rel="stylesheet" href="styles.css" />
   </head>
   <body>
+    <div class="body">
+    <h1 id="hr1">Books > Humor & Entertainment > Humor</h1>
+    <section class="features">
+    
     <img class="main-img" src="images/book-cover.jpg" />
+    <div class="order">
     <h1 class="title">
       How to Talk to Your Cat About Gun Safety: And Abstinence, Drugs, Satanism,
       and Other Dangers That Threaten Their Nine Lives
     </h1>
+    <p id="p1">by <a href="http://#" target="_blank" rel="noopener noreferrer" >Zachary Auburn</a> (Author)</p>
+    <img class="ratings" src="images/ratings.png"> <br>
+
+    <h2 id="hr2">The cats of America are under siege!</h2>
+    <p id="p2">Long gone are the good old days when a cat’s biggest worries were mean dogs or a bath. Modern cats must confront satanists, online predators, the possibility of needing to survive in a post-apocalyptic wasteland, and countless other threats to their nine lives.
+      <br><br>
+      For over four decades, the American Association of Patriots have stood at the vanguard of our country's defense by helping to prepare our nation's cat owners for the difficult conversations they dread having with their pets.
+      <br><br>
+      Written in a simple Q&A format, How to Talk to Your Cat About Gun Safety answers crucial questions such as, “What is the right age to talk to my cat about the proper use of firearms?” and “What are the benefits of my cat living a lifestyle of abstinence?” and especially “Why does my cat need to use the internet? Can’t he just play with yarn like cats used to do?”
+      <br>
+      Source 
+      <a href="http://www.amazon.com/How-Talk-Your-About-Safety-ebook/dp/B01AERZSNC?ref_=ast_author_dp" target="_blank">http://www.amazon.com/How-Talk-Your-About-Safety-ebook/dp/B01AERZSNC?ref_=ast_author_dp</a>
```

Comment:
It's always good to add the source but, for this, let's stick with the design given. So, remove this.